test/Makefile

```
@@ -29,9 +29,23 @@ e2e-test-all: tidy vendor
+	./test/script/olmv1_e2e_setup.sh
+
+olmv1-e2e-run:
+	./test/script/olmv1_e2e_run.sh
+
+olmv1-e2e-upgrade: tidy vendor
+	./test/script/olmv1_e2e_upgrade.sh
+
+olmv1-e2e-cleanup:
+	./test/script/olmv1_e2e_cleanup.sh
+
+olmv1-e2e-test-all: olmv1-e2e-setup olmv1-e2e-run olmv1-e2e-upgrade olmv1-e2e-cleanup
```

checkmake (0.3.2)

[warning] Target "olmv1-e2e-test-all" should be declared PHONY.

(phonydeclared)